tests/test_crash_reports.py

```diff
@@ -50,16 +62,6 @@ def test_that_top_crasher_filter_all_return_results(self, mozwebqa):
         assert cstc.results_count > 0
 
     @pytest.mark.nondestructive
-    @pytest.mark.parametrize(('product'), _expected_products)
-    def test_that_products_page_links_work(self, mozwebqa, product):
-        products_page = ProductsLinksPage(mozwebqa)
```

Comment:
So I take it that this page no longer exists (I couldn't reach it manually). If so, we should also remove the associated page objects.